numpy/linalg/linalg.py

```diff
@@ -37,13 +37,6 @@
     overrides.array_function_dispatch, module='numpy.linalg')
 
 
-# For Python2/3 compatibility
```

Comment:
code runs fine without these, I can't find any use of them so I'm ASSuming it's fine to delete.

Reply:
They were added in f8012ce42e and used locally in this file, but that use was refactored out later. So it seems safe to me.